src/common/containers/GameArea.js

```diff
@@ -234,7 +238,11 @@ export class GameArea extends Component {
     super(props);
 
     if (!props.started) {
-      this.props.history.push('/play');
+      if (props.history.location.pathname.includes('/tutorial')) {
+        props.onStartTutorial();
+      } else {
+        props.history.push('/play');
```

Comment:
Where does this play URL part get consumed?

Reply:
It goes to `containers/Play.js`, which renders the main lobby.